### DISCUSSION TITLE
Adding/Editing Zones

### DISCUSSION THREAD

**Discussion by SupernalAunt** [Techokami/SonicWorldsNext#187] [Q&A] [answered]
I don't know how to change or edit the zones, i'm trying to replace "base zone" with another custom level, but can't find the title cards, and I have no idea how to edit the sprites.

**Comment by Renhoex** [comment, marked as answer]
The sprites are just PNG image files, if you need help with tilesets you should look up the godot document for tilemaps
https://docs.godotengine.org/en/stable/tutorials/2d/using_tilemaps.html

as for name editing, you can find it by selecting the "HUD" node, 
![image](https://github.com/user-attachments/assets/2fa45e2a-9f0a-4c81-a914-88d780f3d1b4)

**Comment by NeonSRB2** [comment]
this is one of the few things the wiki covers actually

https://github.com/Techokami/SonicWorldsNext/wiki/Levels#making-a-new-level
